lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-vm.rb

```diff
@@ -1,5 +1,14 @@
 module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Openstack_CloudManager_Vm < MiqAeServiceVmCloud
     expose :cloud_networks, :association => true
+    expose :cloud_tenant,   :association => true
```

Comment:
@aufi 
Do any of the automate methods need access to the cloud_tenant? It seems the only place you need the cloud tenant is in the https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/openstack/cloud_manager/vm/resize.rb#L43 which is on the model side.
If the automate method doesn't need it we don't have to expose it in the service model.